task3.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Apple pie recipe</title>
+    </head>
+    <body>
+        <h1>Applie pie</h1>
+        <br>
+        <img src="Imagess/apple-pie.jpg" alt="Applie-piee">
+        
+
+        <h2><ins>Total Time</ins></h2>
+        <p>Prep: 20 min. Bake: 45 min.</p>
+        <br>
+        <h2><ins>Makes</ins></h2>
+        <p>8 servings</p>
+        <br>
+        <h2><ins>Ingredients</ins></h2>
+        <ol type="I">
+            <li><p><strong>1/2 cup</strong> sugar</p></li>
+            <li><p><strong>1/2 cup</strong> packed brown sugar</p></li>
+            <li><p><strong>3 tablespoons</strong> all-purpose flour</p></li>
+            <li><p><strong>1 teaspoon</strong> ground cinnamon</p></li>
+            <li><p><strong>1/4 teaspoon</strong> ground ginger</p></li>
+            <li><p><strong>1/4 teaspoon</strong> ground nutmeg</p></li>
+            <li><p><strong>6 to 7 cups</strong> thinly sliced peeled tart apples</p></li>
+            <li><p><strong>1 tablespoon</strong> lemon juice</p></li>
+            <li><p><strong>Pastry for</strong> double-crust pie (9 inches)</p></li>
+            <li><p><strong>1 tablespoon</strong> butter</p></li>
+            <li><p><strong>1 large egg</strong> white</p></li>
+            <li><p><strong>Additional</strong> sugar</p></li>
+        </ol>
+        <br>
+        <h2><ins>Directions</ins></h2>
+        <p>In a small bowl, combine the sugars, flour and spices; set aside. In a large bowl, toss apples with lemon juice. Add sugar mixture; toss to coat. <br>
+            Line a 9-in. pie plate with bottom crust; trim even with edge. Fill with apple mixture; dot with butter. Roll remaining crust to fit top <br> of pie; place over filling. Trim, seal and flute edges. Cut slits in crust.
+            Beat egg white until foamy; brush over crust. Sprinkle with sugar. <br>
+            Cover edges loosely with foil. <strong>Bake at 375° for 25 minutes. Remove foil and bake until crust is golden brown and filling is bubbly, 20-25 minutes longer.</strong> <br>
+            Cool on a wire rack.
+         </p>
+         <br>
+
+         <h3><ins>Nutrition Facts</ins></h3>
+         <p><strong>1 piece: 414 calories, 16g fat (7g saturated fat), 14mg cholesterol, 227mg sodium, 67g carbohydrate (38g sugars, 2g fiber), 3g protein.</strong></p>
+         <br>
+         <p>
+            <em>(c) Tase of Home</em>

```

Comment:
Kod copyright i sličnih tekstova manje važnosti dobro je koristiti `<small>` element.

Također, mali bonus savjet: za copyright simbol postoji takozvani HTML entitet - `&copy;`. Ako ga upišeš u tekst, HTML će entitet pretvoriti u odgovarajući znak, u ovom slučaju &copy;. Popis HTML entiteta možeš vidjeti [ovdje](https://www.freeformatter.com/html-entities.html).

```suggestion
            <small>&copy; Tase of Home</small>
```